lib/linguist/languages.yml

```diff
@@ -3736,6 +3736,15 @@ RenderScript:
   tm_scope: none
   ace_mode: text
   language_id: 323
+Ring:
+  type: programming
+  color: "#0e60e3"
+  extensions:
+  - .ring
+  - .rh
```

Comment:
[I can't find many `.rh` Ring files](https://github.com/search?utf8=%E2%9C%93&q=extension%3Arh+func&type=Code) (those are from a Python dialect it appears).